api/db/schema.prisma

```diff
@@ -37,19 +36,15 @@ model Organization {
 model User {
   id             Int                 @id @default(autoincrement())
   email          String
```

Comment:
Question: Do we want to implement email validation when calling create/update user or is there a way to enforce it here? Also, do we want to use `unique`?

Reply:
yes, we would like to use unique here as well 👍 =1

Reply:
done ✅